tss-esapi/src/error/return_code/tpm/format_zero/error.rs

```diff
@@ -0,0 +1,180 @@
+// Copyright 2022 Contributors to the Parsec project.
+// SPDX-License-Identifier: Apache-2.0
+use crate::{constants::return_code::TpmFormatZeroError, Error, Result};
+use std::convert::TryFrom;
+
+#[derive(Copy, Clone, Debug, PartialEq, Eq, Hash)]
+pub enum TpmFormatZeroErrorResponseCode {
```

Comment:
Similarly - do we need this separate from `TpmFormatZeroError`? I mean, we could keep the type, but make it a wrapper around `TpmFormatZeroError` if you need it for `Display`